python/lsst/daf/butler/_butler.py

```diff
@@ -1584,7 +1588,7 @@ def query_datasets(
             Dataset type object or name to search for.
         collections : collection expression, optional
             A collection name or iterable of collection names to search. If not
-            provided, the default collections are used.  See
+            provided, the default collections are used. Can be a wildcard. See
```

Comment:
Although as written it doesn't do that because I expand the collections before passing to the query system. I need to take another look. Thanks for the reminder.

Reply:
I just tried it without me expanding the collections and the new query system doesn't look like it handles wildcards at all. I'm going to have to query the collections I'm given and then check if the query returns something different to what I started with and then complain if find_first.